### PR TITLE
Added Redis section & general revisions

### PR DESCRIPTION
**Pull request by juliantopley**

### CODE REVIEW

File: _docs/applications/wordpress.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: guide
-title: WordPress Demo
+title: WordPress
 anchor: applications
 module: applications/wordpress
 # Last reviewed by cheryl.hung@storageos.com on 2017-04-12 - commands work fine but can't connect to WP
```
Comment (oicheryl) [inline _docs/applications/wordpress.md]: If you've run through this and it works, can you update?

File: _guides/004-applications.md [hunk 2]
```diff
@@ -13,7 +13,15 @@ sections:
     description: Setting up PostgreSQL with persistent storage.
     module: applications/postgres
   -
-    title: Microsoft SQL
+    title: Microsoft SQL Server
     description: Setting up Microsoft SQL with persistent storage.
     module: applications/mssql
----
+  -
+    title: Redis
```
Comment (oicheryl) [inline _guides/004-applications.md]: I don't see the Redis doc, did you add it to the PR?

File: _guides/004-applications.md [hunk 3]
```diff
@@ -13,7 +13,15 @@ sections:
...
+    description: Setting up Redis with persistent storage.
+    module: applications/redis
+  -
+#    title: WordPress
```
Comment (oicheryl) [inline _guides/004-applications.md]: Let's publish this since it works as a guide to Swarm.

File: _guides/004-applications.md [hunk 4]
```diff
@@ -13,7 +13,15 @@ sections:
...
+#    description: Setting up a Word Press cluster with persistanet storage.
```
Comment (oicheryl) [inline _guides/004-applications.md]: Typo "persistanet"

Comment (oicheryl) [inline _guides/004-applications.md]: "Word Press" should be one word.

File: _guides/004-applications.md [hunk 5]
```diff
@@ -13,7 +13,15 @@ sections:
...
+#    module: explore/wordpress
```
Comment (oicheryl) [inline _guides/004-applications.md]: applications/wordpress

File: _docs/applications/wordpress.md (hunk 1 above)
Comment (juliantopley) [inline _docs/applications/wordpress.md]: I've hashed out the menu item for this as I'm not happy with it right now.  There are a few command line changes with Swarm and I can't see the storage getting provisioned through docker volume ls

File: _guides/004-applications.md (hunk 2 above)
Comment (juliantopley) [inline _guides/004-applications.md]: Should be there - its new content but its not in the repo for some reason...

File: _guides/004-applications.md (hunk 3 above)
Comment (juliantopley) [inline _guides/004-applications.md]: Not yet...

File: _docs/applications/redis.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
+---
+layout: guide
+title: Redis with StorageOS
+anchor: applications
+module: applications/redis
+# Last reviewed by
```
Comment (oicheryl) [inline _docs/applications/redis.md]: by you?

File: _docs/applications/redis.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+title: Redis with StorageOS
+anchor: applications
+module: applications/redis
+# Last reviewed by 
+---
+
+
+# ![image](/images/docs/explore/redislogo.png) Redis with StorageOS
+
+Redis is a popular networked, in-memory, key-value data store with optional durability to disk. The name Redis is derived from 'REmote DIctionary Server'.
```
Comment (oicheryl) [inline _docs/applications/redis.md]: Remove "The name Redis is derived from 'REmote DIctionary Server'."

File: _docs/applications/redis.md [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+   1:M 19 May 08:10:52.256 # Server started, Redis version 3.2.8
+   1:M 19 May 08:10:53.336 * DB loaded from append only file: 1.079 seconds
+   1:M 19 May 08:10:53.336 * The server is now ready to accept connections on port 6379
+   ```
+
+   Refer to the Configuration section below for more details on warning messages.
+
+## Link to redis-benchmark Container
+
+1. link redis-benchmark.
```
Comment (oicheryl) [inline _docs/applications/redis.md]: Capitalise link.

File: _docs/applications/wordpress.md [hunk 9]
```diff
@@ -55,7 +57,7 @@ The first step in this exercise is to get Docker Swarm set up and running on you
        10.245.103.2:2377
    ```
 
-4. Log on to other 2 nodes and join swarm cluster on each
+4. Log on to 2 new nodes and join the Swarm cluster on each
```
Comment (oicheryl) [inline _docs/applications/wordpress.md]: Use "two" instead of "2".

**Pull request merged**